test/potluck_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+require './lib/potluck'
+
+class PotluckTest < Minitest::Test
+
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_is_an_empty_array
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_exists_couscous
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+
+    assert_instance_of Dish, couscous_salad
+  end
+
+  def test_it_exists_cocktail
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+
+    assert_instance_of Dish, cocktail_meatballs
+  end
+
+  def test_it_can_add_couscous_salad
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+    potluck.add_dish(couscous_salad)
+
+    assert_equal [couscous_salad], potluck.dishes
```

Comment:
this is great! love that you started off by asserting that there were no dishes in line 42, and then added a dish, and then asserted that it is now an array with couscous salad  (a dish instance) inside. good progression testing.  you might even consider adding a second one:

```
 def test_it_can_add_dish
    couscous_salad = Dish.new("Couscous Salad", :appetizer)
     fruit_salad = Dish.new("Fruit Salad", :appetizer)

    potluck = Potluck.new("7-13-18")

    assert_equal [], potluck.dishes
    potluck.add_dish(couscous_salad)

    assert_equal [couscous_salad], potluck.dishes

    potluck.add_dish(fruit_salad)

     assert_equal [couscous_salad, fruit_salad], potluck.dishes
end 
```

Strictly speaking, adding the second dish isn't necessary, but in general when writing code, you can sometimes get accidentally lucky in writing a method that works one time-- testing a behavior twice with different data ensures that you didn't get accidentally lucky.